content/en/tracing/service_catalog/guides/integrating-with-ci-visibility.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Integrating with CI Visibility
```

Comment:
Can we update this to "Streamlining the Development Lifecycle with CI Visibility" ? The row in https://docs-staging.datadoghq.com/aaron.hoffman/CIAPP-6849_add-service-catalog-pipelines-docs/tracing/service_catalog/guides/ should be called this too